README.md

```diff
@@ -29,10 +29,10 @@ The lighting pattern is defined as follows.
 
 ## Input and Output
 - input
-  - from eve oss
+  - from [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine)
     - `/autoware_state_machine/lock_state` : Reservation status for on-demand delivery.
```

Comment:
Add a description about values true/false, 1/0, against to the input from the /autoware_state_machine node.